samples/maps/demo/data-class-two-ranges/demo.html

```diff
@@ -1,14 +1,10 @@
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 <script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/annotations.js"></script>
 <script src="https://code.highcharts.com/maps/modules/map.js"></script>
```

Comment:
Can we use `highmaps.js` instead of `highcharts.js` + `map.js`?

Reply:
sure we can, fixed imports